### PR TITLE
more ncr loadouts + crafting recipe standardization + rigbe's mateba

### PR DESCRIPTION
**Pull request by chemistrymain2**
does the same thing the legion loadouts guy did and expands the NCR's weapon options and improves their underpowered stuff

helps equalize them with the legion. NCR and legion now have an = number of automatics and the legion's still faster than the NCR but not four times faster anymore

also standardizes ammo crafting to use blackpowder sheets instead of the reagent

also adds the mateba that rigbe wanted to the game

also adds the pancor jackhammer back to the game and gives it to the NCR cap since he's supposed to be the master chief of the NCR according to rigbe

also buffs the rangemaster since it is currently literally JUST the fucking trail carbine, same ROF and everythign, when the rangemaster is a literal top tier gun
## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by Legate-Maxson** [comment]
as of my deal with Frok, I 100% support this PR

**Comment by chemistrymain2** [comment]
the legion currently has 14 roles with hard yards compared to the NCR's 4

and 12 automatics compared to the NCR's also 4

on top of this, where the nCr moves at a speed of .45 + .4 + .1  = .9 slowdown at all times, the legion, at its slowest, moves at a speed of .4, and for explorers, .23

so i changed it so that the NCR now has 14 roles with hard yards to go with the legion's 14, so now their base slowdown is .55 by default, and cut the service rifle's slowdown to .3, so now it's .4 by default

so your average NCR is still twice as slow as the legion, but instead of the legion being FOUR times faster it's merely twice as fast

also they have 11 automatics to the legion's 12 

they have equal meds to the legion now and also start with armor plates since the legion start with armor plates

the engineer also got some buffs since it's currently a completely ignored role atm and is now better at its job

and the conscript is no longer worse than legion recruits

oh, and also, grenades. like in F:NV the ncr has a lot of the fragmentation (no fire, shrapnel that cuts you, negated by armor) grenades
basically this gives the NCR the decided advantage at range because their guns are either equal to or better than the legion's
but the legion will rip the NCR apart in close range

**Comment by spackorp** [comment]
wholesome chungus pr
40D7AA9C-3418-4B6C-90FE-98944175F722
A2F88E40-55D8-422E-8A5F-B1BA574DEB1D

https://user-images.githubusercontent.com/102334312/163699422-6f8ef1a3-0167-46b7-a4bb-604fa16bc9be.MP4

**Comment by CaputCapital** [comment]
> the legion currently has 14 roles with hard yards compared to the NCR's 4
> 
> and 12 automatics compared to the NCR's also 4
> 
> on top of this, where the nCr moves at a speed of .45 + .4 + .1 = .9 slowdown at all times, the legion, at its slowest, moves at a speed of .4, and for explorers, .23
> 
> so i changed it so that the NCR now has 14 roles with hard yards to go with the legion's 14, so now their base slowdown is .55 by default, and cut the service rifle's slowdown to .3, so now it's .4 by default
> 
> so your average NCR is still twice as slow as the legion, but instead of the legion being FOUR times faster it's merely twice as fast
> 
> also they have 11 automatics to the legion's 12
> 
> they have equal meds to the legion now and also start with armor plates since the legion start with armor plates
> 
> the engineer also got some buffs since it's currently a completely ignored role atm and is now better at its job
> 
> and the conscript is no longer worse than legion recruits
> 
> oh, and also, grenades. like in F:NV the ncr has a lot of the fragmentation (no fire, shrapnel that cuts you, negated by armor) grenades basically this gives the NCR the decided advantage at range because their guns are either equal to or better than the legion's but the legion will rip the NCR apart in close range

This is good. The slowdowns over sand needed to be taken into account for NCR for the longest time. The Pancor Jackhammer for the Captain is reminiscent of the HK CAWs that the Captain had during Outcasts/Early BD Era FO13. The hard yard overhaul on NCR at least allows them more offensive options rather than having to hunker down in the base to be anywhere decent.
